### PR TITLE
chore: rollback CODEOWNER changes

### PR DESCRIPTION
**Pull request by nschonni**
I think some of these might be correct, but this was pushed directly to branch and is creating a lot of notifications.
These should be re-introduced with the agreement of the affected teams

### CODE REVIEW

**Comment by ovflowd** [comment]
What's the issue here? And what do you mean with creating a lot of notifications? 🙃

It will only create notifications when a matching PR to a matching file was changed. Do you have actual examples of people complaining about this?

**Comment by nschonni** [comment]
https://github.com/nodejs/nodejs.org/commit/71bd5b7c2bd50b54fe3e29fa90bff8041ac944a1#commitcomment-105767265 is one, I'm getting notifications for i18n, which isn't what that team is setup for. I think your intentions were good, but you guessed based on team names, not what those teams are actually chartered or normally responsible for without asking them.

I do think the first 2 items you have are correct, but I took the approach that they should be asked before you sign them up for work

PS: I thought branch protection should have prevented you from pushing directly to the branch without a PR. I think admins aren't normally exempted

**Comment by ovflowd** [comment]
Gotcha. Could you keep the changes from line 1-4. I'm quite sure these 2 team assignments are correct.

Regarding the other ones, I believe it was a mistake assigning i18n to translations, the intent was for the configuration files whose kinda belong to the Crowdin Managers which are part of the i18n WG.

But that's debatable, so let's go the safe route and simply keep lines 1-4.

**Comment by ovflowd** [comment]
Yes, I made that commit without a PR as I assumed the CODEOWNERS is a maintainers responsibility, but I legitimate should have double-checked and gone through with a PR.

File: CODEOWNERS [hunk 1]
```diff
@@ -1,44 +1,2 @@
 # Default rules
 * @nodejs/website
```
Comment (nschonni) [inline CODEOWNERS]:
```suggestion
* @nodejs/website

# Node.js Release Blog Posts
/pages/en/blog/release @nodejs/releasers
```

**Comment by tniessen** [comment]
I wasn't even aware that we allow pushing directly at all. But looking at the collaborator guide, it is technically allowed for members of the website team. I wonder if it really should be, or at least strongly discouraged.

**Comment by anonrig** [comment]
> I wasn't even aware that we allow pushing directly at all. But looking at the collaborator guide, it is technically allowed for members of the website team. I wonder if it really should be, or at least strongly discouraged.

I agree with @tniessen. I'm in favor of technically bypassable but a required review process (unless strictly necessary or the change is time sensitive).

**Comment by anonrig** [comment]
Out of topic but somewhat important in this context: Do we really have to include `Website` team to each pull request? I want to continue to be on the team, but I don't want to be notified with every pull request.

**Review by richardlau — APPROVED** [review]

**Comment by ovflowd** [comment]
> I wasn't even aware that we allow pushing directly at all. But looking at the collaborator guide, it is technically allowed for members of the website team. I wonder if it really should be, or at least strongly discouraged.

It is discouraged. Historically speaking, I've done that only with very minor README changes or things by using really good judgment.

**Comment by ovflowd** [comment]
> I agree with @tniessen. I'm in favor of technically bypassable but a required review process (unless strictly necessary or the change is time sensitive).

Usually that's what I've been following. I apologise for any noise and concerns raised here.

**Review by ovflowd — APPROVED** [review]
Thank you for raising this, Nick. LGTM and fast-tracking as this is a time-sensitive change.

**Pull request merged**